packages/orchestration/src/orchestration-types.ts

```diff
@@ -466,6 +487,13 @@ export type AzureContentSafetyFilterReturnType<T extends 'input' | 'output'> =
  */
 export type LlamaGuardFilterConfig = LlamaGuard38BFilterConfig;
 
+/**
+ * Filter return type for Llama Guard.
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export type LlamaGuardFilterReturnType<T extends 'input' | 'output'> =
```

Comment:
Added this here, otherwise the type input would not be used anywhere for Llama guard.

Reply:
I will skip making this verbose and leave it as is.